sdks/java/core/src/main/java/org/apache/beam/sdk/testing/ValueInSingleWindow.java

```diff
@@ -109,7 +109,7 @@ public void encode(ValueInSingleWindow<T> windowedElem, OutputStream outStream,
       T value = valueCoder.decode(inStream, nestedContext);
       Instant timestamp = InstantCoder.of().decode(inStream, nestedContext);
       BoundedWindow window = windowCoder.decode(inStream, nestedContext);
-      PaneInfo pane = PaneInfo.PaneInfoCoder.INSTANCE.decode(inStream, nestedContext);
+      PaneInfo pane = PaneInfo.PaneInfoCoder.INSTANCE.decode(inStream, context);
```

Comment:
Note I only updated `decode` -- `encode` already had the changes. Suspect this was not a bug in practice b/c the `PaneInfoCoder` is not context-sensitive.